### PR TITLE
Fix: Quick temporary fix for float precision

### PR DESCRIPTION
**Pull request by ernestpascual**
## Description of Changes
Temporary fix for float precision

[Link to Jira Ticket](https://halodao.atlassian.net/browse/HDF-420)

### Developer Checklist:

* [x] I have followed the guidelines in our Contributing document
* [x] This PR has a corresponding JIRA ticket
* [x] My branch conforms with our naming convention i.e. `feature/HDF-XXX-description`
* [x] All files have appropriate file headers and documentation
* [ ] I have written new tests for your core changes, as applicable
* [ ] I have successfully ran tests locally

### Reviewers Checklist:
* [ ] Code is readable and understandable; any unclear parts have explanations 
* [ ] UI/UX changes match the corresponding figma/other design resources, if applicable
* [ ] I have successfully ran tests locally

### CODE REVIEW

**Review by schystz — APPROVED** [review]
Looks good after a series of tests

File: src/utils/formatNumber.ts [hunk 1]
```diff
@@ -38,3 +38,13 @@ export function formatNumber(number: number, key?: NumberFormat) {
...
     .toUpperCase()
 }
+
+export function toFixed(num: number, fixed: number) {
+  if (num > 0.00001) {
+    const re = new RegExp('^-?\\d+(?:.\\d{0,' + (fixed || -1) + '})?')
+    const matches = num.toString().match(re)
+    return matches ? matches[0] : num.toString()
+  } else {
+    return '0.0'
```
Comment (schystz) [inline src/utils/formatNumber.ts:48]: @ernestpascual should we just return num here?

**Review by tracyarciaga — APPROVED** [review]
working fine.

File: src/utils/formatNumber.ts (hunk 1 above)
Comment (ernestpascual) [reply src/utils/formatNumber.ts:48]: it will return something if we dont set it to zero. like i have 0.000000000000000006 left, i see 6e

Comment (schystz) [reply src/utils/formatNumber.ts:48]: Ok I guess this will be a limitation for now until we find a long term fix

**Review by mystbrent — APPROVED** [review]

**Pull request merged**